rust/sdk-processor/src/steps/common/processor_status_saver.rs

```diff
@@ -33,15 +34,22 @@ pub fn get_processor_status_saver(
         }
     } else {
         let processor_name = config.processor_config.name().to_string();
-        ProcessorStatusSaverEnum::Default {
-            conn_pool,
-            processor_name,
+        if config.processor_config.is_parquet_processor() {
```

Comment:
we can check if db_config is postgres vs parquet here!

Reply:
yeah remove this and replaced with dbconfig check!